### PR TITLE
PT-1294 | Filter tags + clear filter button

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

- Add clear filters button
- Add filter tags for categories that are not found in keyword sets

### Closes :no_good_woman:

**[PT-1294](https://helsinkisolutionoffice.atlassian.net/browse/PT-1294):**

## Screenshots :camera_flash:

Screenshot 2021-11-25 at 17 40 24

### CODE REVIEW

File: src/domain/events/eventSearchForm/EventSearchForm.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import classNames from 'classnames';
 import { Field, Formik } from 'formik';
-import { Button, IconSearch } from 'hds-react';
+import { Button, IconPen, IconSearch } from 'hds-react';
```
Comment (nikomakela) [inline src/domain/events/eventSearchForm/EventSearchForm.tsx:3]: Needless imports!

**Review by nikomakela — APPROVED** [review]
Tested - it's working

**Review by Jokinen — APPROVED** [review]

File: src/domain/events/filterSummary/FilterSummary.tsx [hunk 2]
```diff
@@ -1,8 +1,11 @@
+import { isArray } from 'lodash';
```
[the remark this answers is not in the document]
Comment (Sartonon) [reply src/domain/events/filterSummary/FilterSummary.tsx:1]: Good catch, this is 100% mistake :D Should have been Array.isArray() :)

File: src/domain/events/EventsSearchPage.tsx [hunk 3]
```diff
@@ -107,6 +116,19 @@ export const useEventsSearch = () => {
...
+    variables;
+
+  const filterActive =
+    !!allOngoingAnd?.length ||
+    !!division?.length ||
+    !!inLanguage?.length ||
+    !!keyword?.length ||
+    !!location ||
+    !!end ||
+    start !== 'now';
```
[the remark this answers is not in the document]
Comment (Sartonon) [reply src/domain/events/EventsSearchPage.tsx:130]: Might be 🤔 To be honest I'm not very happy about how this check is done here but I think the other solution I tried before this was even worse :D Not sure how this should be done so it doesn't break easily

[the remark this answers is not in the document]
Comment (Sartonon) [reply src/domain/events/EventsSearchPage.tsx:130]: Right now I can't think of a better way where you wouldn't need to take this into account when adding new filters

**Pull request merged**